scripts/smoke_tests/commands_test.ps1

```diff
@@ -79,27 +128,104 @@ $commands += "az iot dps enrollment show -g $resource_group_name --dps-name $dps
+$commands += "az dt twin relationship delete-all -g $resource_group_name -n $dt_instance_name --twin-id $dt_twin_id -y"
+$commands += "az dt twin delete -g $resource_group_name -n $dt_instance_name --twin-id $dt_twin_id"
+$commands += "az dt twin delete -g $resource_group_name -n $dt_instance_name --twin-id $dt_target_twin_id"
+$commands += "az dt route delete -g $resource_group_name -n $dt_instance_name --route-name $dt_route_name"
+$commands += "az dt endpoint delete -n $dt_instance_name -g $resource_group_name --en $dt_eventgrid_endpoint -y"
+$commands += "az eventgrid topic delete -g $resource_group_name --name $dt_eventgrid_topic --resource-group $resource_group_name"
+$commands += "az dt reset -g $resource_group_name -n $dt_instance_name -y"
+$commands += "az dt delete -g $resource_group_name -n $dt_instance_name -y"
+
+# IoT Central
+$commands += "az iot central device-template create --app-id $central_app_id --device-template-id $central_device_template_id --content $central_device_template_content"
+$commands += "az iot central device-template show --app-id $central_app_id --device-template-id $central_device_template_id"
+
+$commands += "az iot central device create --app-id $central_app_id --device-id $central_device_id --template $central_device_template_id --simulated"
+# Sleeping to ensure device creation is completed
+$commands += "Start-Sleep -s 30"
+$commands += "az iot central device update --app-id $central_app_id --device-id $central_device_id --device-name $central_device_name"
+$commands += "az iot central device show --app-id $central_app_id --device-id $central_device_id"
+
+$commands += "az iot central device command run --app-id $central_app_id --device-id $central_device_id --command-name $central_device_command --content $central_device_command_payload"
+$commands += "az iot central device command history --app-id $central_app_id --device-id $central_device_id --command-name $central_device_command"
+
+$commands += "az iot central query --app-id $central_app_id --query-string '$central_device_query'"
+$commands += "az iot central export list --app-id $central_app_id"
+$commands += "az iot central role list --app-id $central_app_id"
+$commands += "az iot central user list --app-id $central_app_id"
+
+$commands += "az iot central api-token create --app-id $central_app_id --token-id $central_api_token_id -r 'operator'"
+$commands += "az iot central api-token show --app-id $central_app_id --token-id $central_api_token_id"
+
+$commands += "az iot central organization create --app-id $central_app_id --org-id $central_org"
+$commands += "az iot central organization show --app-id $central_app_id --org-id $central_org"
+
+$commands += "az iot central export destination create --app-id $central_app_id --dest-id $central_export_destination_id --type 'webhook@v1' --name 'Smoke Test Export Destination' --url 'https://www.microsoft.com'"
```

Comment:
Is there anything sensitive passed to the webhook? Or can we use a different destination type?